pkg/kubelet/apis/deviceplugin/v1alpha/api.pb.go

```diff
@@ -820,24 +819,6 @@ func (m *DeviceSpec) MarshalTo(dAtA []byte) (int, error) {
 	return i, nil
 }
 
-func encodeFixed64Api(dAtA []byte, offset int, v uint64) int {
```

Comment:
do we have any tests that ensure previously serialized data decodes correctly and changes like this stay wire-compatible?

Reply:
We have https://k8s-testgrid.appspot.com/sig-release-master-upgrade which I believe are all post-commit. That should be sufficient, right?

Reply:
that's good in one direction, but not exhaustive for all fields (we have a few oddities in our proto generation around nullable type aliases), and doesn't test in the opposite direction (new serialized data is readable by old servers).  I don't see anything concerning in this PR, but it's pretty impossible to eyeball. @smarterclayton, any specific areas we should look at carefully?

Reply:
There are also downgrade tests (https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-master-new-downgrade-cluster) that cover the other direction.

But it's certainly not exhaustive. I'm all in favor of capturing the test coverage gap we think we have here and contributing to get it fixed. Should we open a new issue for that? Should we block this PR?

^ @lavalamp

Reply:
Yeah, that doesn't give me very much additional confidence. It wouldn't catch something that's used even moderately rarely.